content/en/service_management/workflows/private_actions/_index.md

```diff
@@ -20,177 +26,53 @@ further_reading:
 
-{{< tabs >}}
-{{% tab "Docker" %}}
-1. Click **Docker**.
-1. Run the provided `docker run` command on your host, replacing `./config` with the path to the directory you created for the runner configuration.
+## Modes
 
-   You can safely ignore the error `DATADOG TRACER DIAGNOSTIC - Agent Error: connect ECONNREFUSED`.
-{{% /tab %}}
+A private action runner can be used with App Builder, Workflow Automation, or both.
 
-{{% tab "Docker Compose" %}}
-1. Click **Docker Compose**.
-1. Create a `docker-compose.yaml` file and add the provided YAML, or add the `runner` stanza to an existing Docker Compose file. For information on creating a Docker Compose file, see the [official Compose documentation][101].
-1. Replace `./config` with the path to the directory you created for the runner configuration.
-1. Run `docker compose up -d`.
+The following is a general overview diagram for private actions:
 
-   You can safely ignore the error `DATADOG TRACER DIAGNOSTIC - Agent Error: connect ECONNREFUSED`.
+{{< img src="service_management/private_action_runner-diagram_general.png" alt="Overview diagram illustrating how Private actions interact with Datadog and the user's browser" style="width:90%;" >}}
 
-[101]: https://docs.docker.com/compose/compose-application-model/
-{{% /tab %}}
-{{< /tabs >}}
-{{% /collapse-content %}}
+### Mode differences
 
-{{% collapse-content title="App Builder" level="p" %}}
-1. Enter a name for your runner.
-1. Click **App Builder**.
-1. Enter a runner hostname. App Builder calls your runner using this hostname over HTTPS. You must bring your own SSL termination and forward to port 9016 in the container.
-1. Create a directory on your host where the runner can store its configuration, such as `./config`. You can also use this directory to store any credentials required by the runner's connection.
-1. Deploy your runner with Docker or Docker Compose:
-{{< tabs >}}
-{{% tab "Docker" %}}
-1. Click **Docker**.
-1. Run the provided `docker run` command on your host, replacing `./config` with the path to the directory you created for the runner configuration.
+Distinctions between App Builder and Workflows modes include their triggering mechanisms and operational models. App Builder is human-driven, meaning that each action is initiated by user interaction with an App. In contrast, Workflows can run automatically without direct human intervention. Another key difference is that App Builder mode operates through a push model, where actions are triggered by directly accessing a URL on the runner. Conversely, Workflows use a pull model, where the runner periodically checks for tasks to execute. This difference in models results in distinct data handling practices. In Workflows mode, the runner reports the result of private action executions to Datadog. In contrast, App Builder mode does not send your data to Datadog, keeping it in your private environment.
```

Comment:
While I really enjoy the prose you've used here, this paragraph is pretty dense and you're having to use a lot of comparative phrases. Not the end of the world, but have you tried a couple of lists here to define the mode for each product?

Reply:
Tried changing it to a table, because lists were awkward when trying to preserve the comparison factor.